src/bin/cargo_semver.rs

```diff
@@ -257,9 +257,7 @@ fn main() {
         exit_with_error(CliError::new(e, 1), &mut config.shell());
     }
 
-    if env_logger::init().is_err() {
-        eprintln!("ERROR: could not initialize logger");
-    }
+    env_logger::init();
```

Comment:
should be `if let Err(e) = env_logger::try_init().is_err() { eprintln!("ERROR: could not initialize logger: {}", e); }` if you don't want to change behavior (same below)

Reply:
Oh, nice, missed that in the doc, thanks